projects/core/src/icon/icon.stories.ts

```diff
@@ -67,12 +68,102 @@ class AllIcons extends LitElement {
   }
 }
 
+@customElement('demo-icons-collection')
+class CollectionIcons extends LitElement {
+  @state() icons: string[] = [];
+  private collections: any = {};
```

Comment:
This could be a local `const` in the `connectedCallback` method since it's only used there.